### PR TITLE
Spectated Aimbot

### PR DESCRIPTION
**Pull request by LightPower392**
This adds #1021 
Also, this fixes the glitch where the tool-tips could only be one line.

### CODE REVIEW

File: src/hacks/Aimbot.cpp [hunk 1]
```diff
@@ -85,6 +85,40 @@ static settings::Int teammates{ "aimbot.target.teammates", "0" };
...
+{
+    switch (*specmode)
+    {
+    // Always on
+    default:
+    case 0:
+        break;
+        // Disable if being spectated in first person
+    case 1:
+        if (g_pLocalPlayer->spectator_state == g_pLocalPlayer->FIRSTPERSON)
```
Comment (BenCat07) [inline src/hacks/Aimbot.cpp:106]: This if statement will only affect enable = *specenable, not the rest.

File: src/hacks/Aimbot.cpp [hunk 2]
```diff
@@ -85,6 +85,40 @@ static settings::Int teammates{ "aimbot.target.teammates", "0" };
...
+            enable = *specenable;
+            slow_aim = *specslow;
+            fov = *specfov;
+            return;
+        break;
+        // Disable if being spectated
+    case 2:
+        if (g_pLocalPlayer->spectator_state != g_pLocalPlayer->NONE)
```
Comment (BenCat07) [inline src/hacks/Aimbot.cpp:114]: Once again, wrong indentation under the assumption that everything will be affected by the if statement.

File: src/hacks/Aimbot.cpp [hunk 3]
```diff
@@ -85,6 +85,40 @@ static settings::Int teammates{ "aimbot.target.teammates", "0" };
...
+    default:
+    case 0:
+        break;
+        // Disable if being spectated in first person
+    case 1:
+        if (g_pLocalPlayer->spectator_state == g_pLocalPlayer->FIRSTPERSON)
+            enable = *specenable;
+            slow_aim = *specslow;
+            fov = *specfov;
+            return;
```
Comment (BenCat07) [inline src/hacks/Aimbot.cpp:110]: The return is unneccessary with a break statement

File: src/hacks/Aimbot.cpp [hunk 4]
```diff
@@ -85,6 +85,40 @@ static settings::Int teammates{ "aimbot.target.teammates", "0" };
...
+        break;
+        // Disable if being spectated
+    case 2:
+        if (g_pLocalPlayer->spectator_state != g_pLocalPlayer->NONE)
+            enable = *specenable;
+            slow_aim = *specslow;
+            fov = *specfov;
+            return;
```
Comment (BenCat07) [inline src/hacks/Aimbot.cpp:118]: The return statement is unneccesary with a break statement

File: src/hacks/Aimbot.cpp [hunk 5]
```diff
@@ -85,6 +85,40 @@ static settings::Int teammates{ "aimbot.target.teammates", "0" };
...
+    };
+    return;
```
Comment (BenCat07) [inline src/hacks/Aimbot.cpp:120]: This return is also unneccesary

**Comment by LightPower392** [comment]
updated

**Comment by BenCat07** [comment]
Let me test this first

**Pull request merged**